paperqa/llms.py

```diff
@@ -205,16 +207,17 @@ async def execute(
                 result.prompt_count = sum(
                     self.count_tokens(m["content"]) for m in messages
                 ) + sum(self.count_tokens(m["role"]) for m in messages)
+                usage = (0, 0)
                 if callbacks is None:
-                    output = await self.achat(messages)
+                    output, usage = await self.achat(messages)
                 else:
                     sync_callbacks = [
                         f for f in callbacks if not is_coroutine_callable(f)
                     ]
                     async_callbacks = [f for f in callbacks if is_coroutine_callable(f)]
                     completion = self.achat_iter(messages)
                     text_result = []
-                    async for chunk in completion:  # type: ignore[attr-defined]
+                    async for chunk, _usage in completion:  # type: ignore[attr-defined]
```

Comment:
Why is this `_usage` if it's used below? Also, imo `usage` is a bit vague out of context.

Do you mind naming it something like `chunk_usage`